_includes/v22.1/known-limitations/row-level-ttl-limitations.md

```diff
@@ -0,0 +1,2 @@
+- You cannot use [foreign keys](foreign-key.html) to create references to or from a table that uses Row-Level TTL.
+- [`SELECT` queries](selection-queries.html) against tables with Row-Level TTL enabled do not filter out expired rows from the result set. This feature may be added in a future release. For now, follow the instructions in [Filter out expired rows from a selection query](row-level-ttl.html#filter-out-expired-rows-from-a-selection-query).
```

Comment:
i would also probably add the following (if we care about performance):

* We don't execute the deletes to be close to the range, e.g. something from europe-west may delete from us-east.
* Deletes do not make use of any available index on crdb_internal_expiration which would speed things up.
* There is no way to customize a delete expiration yet

Reply:
thanks Oliver, updated this a bit based on that info

the only one I left out is "customize a delete expiration yet" - do you mind explaining more about what that means?  do you mean customizing the delete expiration per row (somehow) instead of setting it for the whole table using the `ttl_expire_after`?